bleak/backends/winrt/client.py

```diff
@@ -526,6 +526,9 @@ async def get_services(self, **kwargs) -> BleakGATTServiceCollection:
                 "Could not get GATT services",
             )
 
+            if not self.is_connected:
```

Comment:
I think this needs to be at the beginning of the function, otherwise `self._requester` could be `None` above.